src/main/java/com/volasoftware/tinder/constants/MailConstant.java

```diff
@@ -0,0 +1,11 @@
+package com.volasoftware.tinder.constants;
+
+public class MailConstant {
```

Comment:
Are the constants real for mail?

Reply:
Without more requirements and further context about it, I think they are. What exactly do you mean?

Reply:
https://github.com/tsenovi/spring-tinder-application/pull/16/commits/4632b3329a2080234175c7b0c4f262cf9dadc9b7